mackerel-plugin-mongodb/lib/mongodb.go

```diff
@@ -183,12 +184,18 @@ type MongoDBPlugin struct {
 }
 
 func (m MongoDBPlugin) fetchStatus() (bson.M, error) {
-	session, err := mgo.Dial(m.URL)
+	mongoDBDialInfo := &mgo.DialInfo{
+		Addrs:    []string{m.URL},
+		Direct: true,
```

Comment:
Please apply `gofmt`. (We should check it in ci...)
```diff
diff --git a/mackerel-plugin-mongodb/lib/mongodb.go b/mackerel-plugin-mongodb/lib/mongodb.go
index bcc31f0..d63459e 100644
--- a/mackerel-plugin-mongodb/lib/mongodb.go
+++ b/mackerel-plugin-mongodb/lib/mongodb.go
@@ -185,9 +185,9 @@ type MongoDBPlugin struct {

 func (m MongoDBPlugin) fetchStatus() (bson.M, error) {
        mongoDBDialInfo := &mgo.DialInfo{
-               Addrs:    []string{m.URL},
-               Direct: true,
-               Timeout:  10 * time.Second,
+               Addrs:   []string{m.URL},
+               Direct:  true,
+               Timeout: 10 * time.Second,
        }
        session, err := mgo.DialWithInfo(mongoDBDialInfo)
        if err != nil {
```

Reply:
@astj 
I have been apply gofmt.